src/components/approval-type/ApprovalType.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div id="approval-type">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label id="approval-type-label">Approval Type</label>
+      </v-col>
+      <v-col cols="12" sm="9">
+        <v-radio-group class="payment-group" v-model="approvalTypeSelected" @change="radioButtonChanged">
+          <!-- COURT ORDER radio button -->
+          <v-radio id="court-order-radio" class="mb-0"
+             :label="getRadioText(ApprovalTypes.VIA_COURT_ORDER)"
+             :value="ApprovalTypes.VIA_COURT_ORDER"
+          />
+          <v-form ref="courtNumRef" id="court-num-form" v-model="valid" class="mt-4 ml-8">
+            <v-text-field
+              @input="courtOrderNumberChanged"
+              id="court-order-number-input"
+              v-model="courtOrderNumber"
+              label="Court Order Number"
+              :rules="courtOrderNumRules"
+              :disabled="approvalTypeSelected === ApprovalTypes.VIA_REGISTRAR"
+              @update:error="emitValidationError"
```

Comment:
I prefer to use the full syntax for the event handler, so it's clear that it's a method and not a property, ie, `emitValidationError()`.

The downside is that you have to explicitly pass the event data (`$event`), which is inherent in the first synatax.

Not a deal-breaker.

Reply:
Good call, will fix

Reply:
Thanks for update.

Same thing on line 8 actually.